src/system-fetch.js

```diff
@@ -66,9 +66,13 @@
       else
         url = url.substr(7);
       return fs.readFile(url, function(err, data) {
-        if (err)
-          return reject(err);
-        else {
+        if (err) {
+          if (err.errno == 34) {
```

Comment:
I couldn't find anything specifically. Googling for "node enoent" and variants is a flood of "I don't understand this error" rather than something about the message/stack thing. I did find this:

http://neversaw.us/nodejs-api/errors.html#errors_system_errors

Which clarified that ENOENT &co are simply augmented Error objects. Also clarified: the error.stack getter returns a string when it's called. (Foolish I was expecting an array.)

Based on the Error constructor, I suspect that (some versions) of Node or NPM have a bug in their construction of system errors, and they sub in the message for the stack.

As to testing: I specifically added tests about the non-existent file to ensure that it had a proper message and stack. Testing for e.g. errno 34  IMO is testing the implementation.